template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/WebApi-CSharp/Areas/IdentityService/Data/Migrations/IdentityServiceDbContextModelSnapshot.cs

```diff
@@ -0,0 +1,366 @@
+﻿// <auto-generated />
+using System;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Infrastructure;
+using Microsoft.EntityFrameworkCore.Metadata;
+using Microsoft.EntityFrameworkCore.Migrations;
+using Company.WebApplication1.Identity.Data;
+
+namespace Company.WebApplication1.Identity.Data.Migrations
+{
+    [DbContext(typeof(IdentityServiceDbContext))]
+    partial class IdentityServiceDbContextModelSnapshot : ModelSnapshot
+    {
+        protected override void BuildModel(ModelBuilder modelBuilder)
+        {
+            modelBuilder
+#if (UseLocalDB)
+                .HasAnnotation("ProductVersion", "2.0.0-preview1-24567")
```

Comment:
Would be nice to get these updated before release... how do we normally do that? Or did we not ever need to?

cc @bricelam in case you have thoughts.